services/web/server/tests/login/test_storage.py

```diff
@@ -42,41 +42,76 @@ def storage_server(loop, aiohttp_server, app_cfg):
         assert "user_id" in query
 
         assert query["user_id"], "Expected user id"
+
         return web.json_response({
             'data': [{"user_id": int(query["user_id"])}, ]
         })
 
+    async def _get_filtered_list(request: web.Request):
+        assert not request.has_body
 
-    app.router.add_get("/v0/locations", _get_locs)
-    app.router.add_get("/v0/locations/0/files/{file_id}", _get_dlink)
-
-    assert cfg['host']=='localhost'
-
-    server = loop.run_until_complete(aiohttp_server(app, port= cfg['port']))
-    return server
+        query = request.query
+        assert query
+        assert "user_id" in query
 
-@pytest.mark.travis
-async def test_storage_locations(client, storage_server):
-    url = "/v0/storage/locations"
+        assert query["user_id"], "Expected user id"
+        assert query["uuid_filter"], "expected a filter"
 
-    async with LoggedUser(client) as user:
-        print("Logged user:", user) # TODO: can use in the test
+        return web.json_response({
+            'data': [{"user_id": int(query["user_id"]), "uuid_filter": str(query["uuid_filter"])} ]
+        })
 
-        resp = await client.get(url)
-        payload = await resp.json()
-        assert resp.status == 200, str(payload)
 
-        data, error = unwrap_envelope(payload)
+    #app.router.add_get("/v0/locations", _get_locs)
+    #app.router.add_get("/v0/locations/0/files/{file_id}", _get_dlink)
+    app.router.add_get("/v0/locations/0/files/metadata", _get_filtered_list)
 
-        assert len(data) == 1
-        assert not error
+    assert cfg['host']=='localhost'
 
-        assert data[0]['user_id'] == user['id']
+    server = loop.run_until_complete(aiohttp_server(app, port= cfg['port']))
+    return server
 
+#@pytest.mark.travis
```

Comment:
pls mark them as ``pytest.mark.skip`` and state a reason ... or simply delete them ... otherwise they get forgotten